app/controllers/katello/api/v2/upstream_subscriptions_controller.rb

```diff
@@ -35,7 +35,7 @@ def index
       N_("Update the quantity of one or more subscriptions on an upstream allocation")
     param :organization_id, :number, :desc => N_("Organization ID"), :required => true
     param :pools, Array, desc: N_("Array of Pools to be updated. Only pools originating upstream are accepted."), required: true do
-      param :id, String, desc: N_("ID of local pool to update"), required: true
+      param :id, String, desc: N_("ID of local pool to update (use Katello ID)"), required: true
```

Comment:
What do you think about saying "Katello UD of local pool to update" and similar for the other apidoc you've updated? I think it's a little more clear and probably more easily translated

Reply:
There are also a few placed in this controller where we say 'local' maybe we should just replace them all with Katello to be super-duper-clear?